class.jetpack-debugger.php

```diff
@@ -18,6 +18,12 @@ private static function is_jetpack_support_open() {
 		}
 	}
 
+	private static function what_jetpack_plan() {
+		$plan = Jetpack::get_active_plan();
+		$plan = $plan['class'];
```

Comment:
Should we add a fallback value if `$plan['class']` is empty?  Like if someone adds a new plan down the road and forgets to include a class?